tests/integration/test_storage_rabbitmq/test_failed_connection.py

```diff
@@ -44,29 +46,120 @@
 # Helpers
 
 
-def suspend_rabbitmq(rabbitmq_cluster):
+class RabbitMQMonitor:
```

Comment:
Should we start monitor only for specific tests? Currently it seems on for all tests

Reply:
Ah, ok I see, I missed that there are only two tests in this file, then it is ok.